src/components/content/catalog/services/update/UpdateService.tsx

```diff
@@ -165,14 +170,18 @@ function UpdateService({
         return false;
     };
 
+    const onChange = (e: CheckboxChangeEvent) => {
+        setIsRemoveServiceTemplateUntilApproved(e.target.checked);
+    };
+
     return (
         <div className={catalogStyles.updateUnregisterBtnClass}>
```

Comment:
result text shown is wrong. In this specific case, the service was submitted for review. It was not auto-approved. 

![image](https://github.com/user-attachments/assets/b80bc68b-91ee-449c-94a6-ff3c34c19f00)